src/main/java/core/basesyntax/FigCol.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class FigCol implements Figure {
+    private Color color;
+
+    public FigCol(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+}
```

Comment:
The class name 'FigCol' uses an abbreviation and doesn't fully convey the purpose of the class. Consider renaming it to a more descriptive name that avoids abbreviations. Additionally, avoid using abstract classes to set behavior for classes. If 'FigCol' is intended to provide common functionality for figures with a color, consider using an interface for defining the contract and a concrete class for shared behavior.